lib/luminork-server/src/service/v1/common.rs

```diff
@@ -62,7 +63,17 @@ impl IntoResponse for ApiError {
     fn into_response(self) -> Response {
         let status =
             StatusCode::from_u16(self.status_code).unwrap_or(StatusCode::INTERNAL_SERVER_ERROR);
+
+        // Store message before moving self
+        let message = self.message.clone();
```

Comment:
Is there any reason why we don't do something similar in SDF or does this point to a difference between how the projects are setup?